db/schema.rb

```diff
@@ -258,7 +258,7 @@
     t.index ["status"], name: "index_repositories_on_status"
   end
 
-  create_table "repository_dependencies", id: :serial, force: :cascade do |t|
+  create_table "repository_dependencies", id: :bigint, default: -> { "nextval('repository_dependencies_new_id_seq6'::regclass)" }, force: :cascade do |t|
```

Comment:
I don't think we should have the hardcoded sequence name as default, so trying to fix this now.